{
    "checksum": "1b36e452b8bdf289ad5d5fa8c1840f5bb8f884d36088c0c386a7e72c46d5daaf",
    "file": "serviceerrors.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 2,
    "package": "python-jtoolkit",
    "path": "python-jtoolkit/0.7.8-4/jToolkit/serviceerrors.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 2536,
        "license": "/copyright/license/python-jtoolkit/0.7.8-4/",
        "metric": {
            "size": 1436
        },
        "pts_link": "https://tracker.debian.org/pkg/python-jtoolkit",
        "sloc": [
            [
                "python",
                15143
            ],
            [
                "makefile",
                16
            ]
        ],
        "suites": [
            "lenny"
        ]
    },
    "raw_url": "/data/main/p/python-jtoolkit/0.7.8-4/jToolkit/serviceerrors.py",
    "stat": {
        "perms": "rwxr-xr-x",
        "size": 1954,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "0.7.8-4"
}